src/ngraph/op/constant.cpp

```diff
@@ -53,9 +53,6 @@ vector<string> op::Constant::get_value_strings() const
 {
     vector<string> rc;
 
```

Comment:
Let's add a comment here and below to indicate that compiler flags are set via cmake so that three months from now it's not mysterious what's going on.